src/acceptance-testing/release-readiness.md

```diff
@@ -0,0 +1,125 @@
+# Release Readiness Process (RRP)
+
+## Overview
+
+This document defines the process and expectations for devnet releases in the OP Stack. It establishes a consistent framework for determining when a devnet is ready for release, how results are communicated, and how pass/fail determinations are made. By following these procedures, we can ensure that devnets meet quality standards before promotion, reducing the risk of critical bugs reaching production networks.
+
+## Roles and Responsibilities
+
+While the Platforms team serves as the primary custodian of this release readiness process, its success relies on collaborative ownership with Protocol and contributions from across the organization.
+
+| Role | Responsibilities |
+|------|-----------------|
+| **Platforms Team** | • Maintain the Release Readiness Process (RRP)<br>• Run acceptance tests<br>• Make final pass/fail determinations<br>• Communicate test results to stakeholders |
+| **Feature Teams** | • Write acceptance tests for specific features<br>• Run feature-specific tests<br>• Fix identified issues in their features<br>• Provide input on severity of identified issues |
+| **Release Coordinator** | • Coordinate the overall release process<br>• Track progress against the Release Readiness Checklist<br>• Facilitate communication between teams<br>• Document release decisions and rationales |
+
+## Objectives
+
+The primary objectives of the Devnet Release Readiness process are:
+
+1. Release production networks without critical bugs
+2. Ensure high feature coverage through comprehensive testing
+3. Establish a clear process for devnet promotion decisions
+4. Provide visibility into the release readiness of devnets
+5. Define responsibilities for teams involved in the release process
+
+## Release Readiness Process
+
+### Prerequisites
+
+Before a devnet can be considered for release, the following prerequisites must be met:
+
+1. All required features must be deployed to the devnet
+2. A Release Readiness Checklist (RRC) must be defined for the devnet
```

Comment:
What does this actually mean?

Reply:
To phrase them differently:

Before we release (make public) a devnet we should have first done two things:
1. Implemented and deployed the features it intends to test/showcase
2. Defined a set of acceptance tests we'll run to test these features (from #1)

Does that make sense?

Mind you, I've changed this in the latest revision to be:
> Before a devnet can be considered for release, the following prerequisites must be met:
> 
> 1. All new features must have acceptance testing coverage in [op-acceptance-tests](https://github.com/ethereum-optimism/optimism/tree/main/op-acceptance-tests)
> 2. The acceptance tests, as defined by the Release Readiness Checklist, should be passing on a local kurtosis-based devnet

This new definition is simpler, yet stricter.

Thoughts?